releasenotes/notes/v05-encoding-pack-str-error-handle-d11960b72472af7c.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: Resolves an issue where v0.5 encoding endpoint errors were output to ``stderr`` instead of
+    to a logger if non-text span tags were set.
```

Comment:
Could we simplify the release note it's a bit wordy/confusing. Maybe something like this would be better `tracing: Ensure encoding errors are logged`

Also does this change require a release note? MsgPack encoders are an internal component and this fix is for an encoder that is disabled by default.

Reply:
Simplified the release note, but I think this change does require a release note as users previously wouldn't have seen encoding errors from their logs, they would have had to dig into their application's stderr directly.